packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -421,6 +421,47 @@ void main() {
     expect(tileNode.hasPrimaryFocus, isTrue);
   });
 
+    testWidgets('CheckboxListTile can be disabled', (WidgetTester tester) async {
+      bool? value = false;
+      bool enabled = true;
+
+      await tester.pumpWidget(
+        Material(
+          child: StatefulBuilder(
+            builder: (BuildContext context, StateSetter setState) {
+              return wrap(
+                child: CheckboxListTile(
+                  title: const Text('Title'),
+                  enabled: enabled,
+                  value: value,
+                  onChanged: (bool? v) {
+                    setState(() {
+                      value = v;
+                      enabled = !enabled;
+                    });
+                  },
+                ),
+              );
+            },
+          ),
+        ),
+      );
+
+      final Finder checkbox = find.byType(Checkbox);
+      /// verify initial values
```

Comment:
nit: I think maybe double slash // here would be better? Got some hints from [Style Guide for Flutter](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#:~:text=for%20more%20details.-,Use%20///%20for%20public%2Dquality%20private%20documentation,-In%20general%2C%20private):)

Reply:
Thanks done